presto-pinot-toolkit/src/test/java/com/facebook/presto/pinot/query/TestPinotQueryGenerator.java

```diff
@@ -366,6 +367,8 @@ public void testSimpleSelectWithTopN()
                 "SELECT regionId, city, fare FROM realtimeOnly ORDER BY fare, city DESC LIMIT 50",
                 defaultSessionHolder,
                 ImmutableMap.of());
+        ProjectNode projectNode = project(planBuilder, topnFareAndCity, ImmutableList.of("regionid", "city"));
```

Comment:
What about Pinot TOP queries (ie aggregation-order-by-limit-desc queries). Do that change too with this PR ?

Reply:
For PQL we don't allow this (ordering on aggregated data) to be pushed down and we do topN operation in Presto so not impacted.

For SQL this is allowed and there are test cases which are passing. With aggregation it is not syntactically supported that a order by column doesn't appear in final selections so no need to add that test case.